src/main/java/core/dao/ManufacturerDao.java

```diff
@@ -0,0 +1,19 @@
+package core.dao;
+
+import core.model.Manufacturer;
+import java.util.List;
+import java.util.Optional;
+
+public interface ManufacturerDao {
+    Manufacturer add(Manufacturer manufacturer);
+    
+    Optional<Manufacturer> get(Long id);
+    
+    List<Manufacturer> getAll();
+    
+    Manufacturer update(Long id, Manufacturer manufacturer);
```

Comment:
```suggestion
    Manufacturer update(Manufacturer manufacturer);
```
id is present in the model

Reply:
ok, i see